docs/docs/20-usage/20-workflow-syntax.md

```diff
@@ -1,6 +1,12 @@
 # Workflow syntax
 
-The workflow section defines a list of steps to build, test and deploy your code. Steps are executed serially, in the order in which they are defined. If a step returns a non-zero exit code, the workflow and therefore all other workflows and the pipeline immediately aborts and returns a failure status.
+The workflow section defines a list of steps to build, test and deploy your code. Steps are executed serially, in the order in which they are defined.
+
+If a step returns a non-zero exit code, the workflow and therefore the whole pipeline immediately aborts and returns a failure status.
```

Comment:
```suggestion
The Workflow section defines a list of stepsto build, test and deploy your code. The steps are executed serially in the order in which they are defined. If a step returns a non-zero exit code, the workflow and therefore the entire pipeline terminates immediately and returns an error status.
```

Reply:
That has some typos in it, like `stepsto`

Reply:
Fixed